Libplanet/Blockchain/Renderers/DelayedActionRenderer.cs

```diff
@@ -244,6 +254,27 @@ public void RenderBlockEnd(Block<T> oldTip, Block<T> newTip)
 
             _localRenderBuffer.Value = new Dictionary<HashDigest<SHA256>, List<ActionEvaluation>>();
             DiscoverBlock(oldTip, newTip);
+
+            if (_reorgResistantHeight == 0)
+            {
+                return;
+            }
+
+            var quotient = Math.DivRem(newTip.Index, _reorgResistantHeight, out _);
```

Comment:
`quotient` seems not used elsewhere. How about just check `newTip.Index` < `_reorgResistantHeight` instead of `quotient` <= 0?

Reply:
Previously I coded cleaning buffer every `reorgResistantHeight` as you say. That's why calculate `quotient`.
And yes, you're right. it working exactly same.

Reply:
In my opinion, clear every interval looks better because clearing needs array iteration.

Reply:
IMHO, clearing buffer per each render isn't a big matter because array iteration overhead isn't heavy.